src/pet.js

```diff
@@ -0,0 +1,83 @@
+
+const vPet = {
+    minAge: 0,
+    maxAge: 30,
+    minHunger: 0,
+    maxHunger: 10,
+    minFitness: 0,
+    maxFitness: 10,
+    errorMessage: 'Your pet is no longer alive :('
+}
+
+class Pet {
+    constructor (name) {
+        this.name = name;
+        this.age = vPet.minAge;
+        this.hunger = vPet.minHunger;
+        this.fitness = vPet.maxFitness;
+        this.children = [];
+    }
+
+    get isAlive () {
+        return this.fitness > vPet.minFitness && this.hunger < vPet.maxHunger && this.age < vPet.maxAge;
+    }
+   
+    growUp () {
+        if (!this.isAlive) throw new Error (vPet.errorMessage);
+
+        this.age ++;
+        this.hunger += 5;
+        this.fitness -= 3;
+    }
+
+    walk () {
+        if (!this.isAlive) throw new Error (vPet.errorMessage);
+        
+        if ((this.fitness + 4) <= vPet.maxFitness) {
+            this.fitness += 4;
+        } else {
+            this.fitness = vPet.maxFitness;
```

Comment:
It's really good to reference one point of truth for your default values, so that if they ever change you only need to update your code in one place. +1